packages/zkdb/src/storage-engine/base.ts

```diff
@@ -108,6 +108,11 @@ export abstract class StorageEngineBase<T, K, M> {
    */
   public abstract readFile(_path: string): Promise<Uint8Array>;
 
+  /**
+   * Clean up the path
+   */
+  public abstract cleanUp(): Promise<boolean>
```

Comment:
I think `cleanUp()` shouldn't be a part of `MerkleTreeBase` if it was a non-standard method that uses for testing.

Reply:
`cleanUp()` is a part of the `StorageEngine`. Did you mean `StorageEngine`?

Reply:
> `cleanUp()` is a part of the `StorageEngine`. Did you mean `StorageEngine`?

Ah! yes. I think we shouldn't implement `cleanUp()` just to make sure it won't be used incorrectly.

Reply:
I removed this method. Added the clean up to the test file